### PR TITLE
Fix formatting of incomplete CSS value comments

### LINKED ISSUE #15948: [css]: custom property with comment breaks
**Prettier 3.2.3**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEALAjAAmAHSpzAWkICMBDMAawHMAnCAVygBMlMSAbBuAbjwOIB5AHJsAllDEwxZDpgD0AKkwjMi+X3xFCggGK62SlfrUb+2sAwDOMCAFtCAB3qO2ANzK0AFEP0BKPABfEAAaEAhHaWgrZFBPegB3AAVPBBiUWQSyAE8YsJJaCko4GABlMjs4ABkJOGQAM1krOHzCqhLSxwoJamQYWm4w5rsxPoGWkDgAD0c4WjFK2FkAFTmoTzE4dMaOZqGejjgARQYIeAamiYArKynSg+PT86QdvZAARye4JJd0kDIrIQoHA4MxQaEQP0yGIOD0AML2OxkZD-DgcCFWB4AQRg-TEJAY8CScxqwIuuwmqBgdg4AHVUFItl0wHBSmkpGI3FJsiiwFY8iA3NwAJIsBBlMDzSJYlilGDZQ7kt7OCDNWmFRwo5xbOZuOphCTNWgwH5kahIpUTLq0I28-kQ5wSGC0sTMGCoZAADgADGFaHBPmJ-abzciXpcwjAyCQXW6PUgAExhaxwZbR7YRyZ2EigsHMKpkKDUBhmuC6CC0JG4noosiEiAgQKBIA)

```sh
--parser css
```

**Input:**

```css
h1 {
  --background: blue;
  --ON: initial /* ON */;
  --OFF: /* OFF */;
  --custom-prop: var(--OFF)
}
```

**Output:**

```css
h1 {
  --background: blue;
  --ON: initial /* ON */;
  --OFF:  /* OFF *;
  --custom-prop: var(--OFF);
}

```

**Expected output:**

```css
h1 {
  --background: blue;
  --ON: initial /* ON */;
  --OFF: /* OFF */;
  --custom-prop: var(--OFF);
}
```

**Why?**
In this case, the custom property without a value is indented to a point where the ending slash (of the --OFF property comment) gets removed and opens up the comment, essentially commenting out the following code.

### PR DESCRIPTION
**Pull request by sosukesuzuki**
## Description

Fixes #15948

This is a non-ideal workaround, because it's a parser bug.

## Checklist

- [x] I’ve added tests to confirm my change works.
- [x] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by fisker** [comment]
This doesn't fix the problem,

**Prettier pr-16583**
[Playground link](https://deploy-preview-16583--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEALAjAAmAHSpzAWkIHkAxMpAgegCpNyzNbqBuPA4xqm+x5thyKkKPXgwoD2+YdwLy6Epi2mcRleQr6SVQrqM1alUvAF88IADQgIABxgBLaAGdkoAIYAnTxADuABS8EVxR3ABtfdwBPV2sAI093MABrOBgAZXcAWzgAGQcoOGQAM3DnOHjElLT02ySCgHNkGE8AVwqQcqyHZraOuAAPWzhPBxzYcIAVEagvBzgQ0rDy62dGsLgARVaIeBKyjoArZwH09a2dvaQllZAAR0u4fx9bEJB3Z0JCuAATX6sQC13A4wo0AMIQLJZdzId5hMIAtZQBobACCMBaDjirXg-hG+UK+2WHVQMCyYQA6qgHPBnHUwHB0sEaQ4AG40qKwsDOWIgVntACSUD+sHSYFG9lRwvSMCiGyJt1sPnKFMStlhSoWI1ZRWsBXKnhgz3cDWhCo6dU8Bq5PIBSoKMApDh+MFQyAAHAAGayeOAPBy+42mmHXA7WGDuOJOl1upAAJmsrXKk0jizDIDgWTivz+P1y7mRrRNcDIEE80IxjVh7hxEBAplMQA)

```sh
--parser css
```

**Input:**

```css
h1 {
  --OFF:  /* OFF */;
  --OFF:   /* OFF */;
  --OFF:    /* OFF */;
  --OFF:     /* OFF */;
  --OFF:      /* OFF */;
  --OFF:       /* OFF */;
}

```

**Output:**

```css
h1 {
  --OFF: /*  /* OFF * */;
  --OFF: /*   /* OFF  */;
  --OFF: /*    /* OFF */;
  --OFF: /*     /* OF */;
  --OFF: /*      /* O */;
  --OFF: /*       /*  */;
}

```

**Comment by fisker** [comment]
I'll take a look.

File: tests/format/css/comments/__snapshots__/format.test.js.snap [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+  --OFF: /* OFF */;
+  --OFF:  /* OFF */;
+  --OFF:   /* OFF */;
+  --OFF:    /* OFF */;
+  --OFF:     /* OFF */;
+  --OFF:      /* OFF */;
+  --OFF:       /* OFF */;
+  --OFF:        /* OFF */;
+  --OFF:         /* OFF */;
+  --OFF:          /* OFF */;
```
Comment (fisker) [inline tests/format/css/comments/__snapshots__/format.test.js.snap:19]: I can't find specs for how this should work, not sure if it can be removed.

Comment (fisker) [reply tests/format/css/comments/__snapshots__/format.test.js.snap:19]: @alexander-akait Maybe you know about it?

**Review by sosukesuzuki — COMMENTED** [review]
This is my PR so I cannot approve it on GitHub, but LGTM.

**Pull request merged**